conda/cli/conda_argparse.py

```diff
@@ -115,6 +115,12 @@ def do_call(args, parser):
     """
     # First, check if this is a plugin subcommand; if this attribute is present then it is
     if getattr(args, "plugin_subcommand", None):
+        # Run the pre_command actions
+        actions = context.plugin_manager.get_pre_command_hook_actions(
+            args.plugin_subcommand.name, args
+        )
+        tuple(action() for action in actions)
```

Comment:
```suggestion
        # Run the pre_command actions
        _run_pre_commands(args.plugin_subcommand.name, args)
```

Reply:
Important bit is that you hadn't passed command name and args, and there is a bit of repetition going on that we can move into `_run_pre_commands` to be able to have just one code path for running the pre commands here.